src/helpers.ts

```diff
@@ -0,0 +1,55 @@
+import { ActionInput, ActionInputParam, DeploymentEnvironment } from './types'
+import * as core from '@actions/core'
+
+export const getActionInputs = (variables: ActionInputParam[]): ActionInput => {
+  return variables.reduce((obj, variable) => {
+    let value: string | undefined = core.getInput(
+      variable.name,
+      variable.options
+    )
+    if (!value) {
+      if (Object.hasOwn(variable, 'default')) {
+        value = variable.default
+      }
+    }
+    return Object.assign(obj, { [variable.name]: value })
+  }, {}) as ActionInput
+}
+
+export const getOrganisationalUnits = (
+  labels: string
+): DeploymentEnvironment[] => {
+  if (labels === undefined || labels === '') {
+    return []
+  }
+
+  const environmentStrings = labels.split(',')
+
+  const environmentArray: DeploymentEnvironment[] = []
+
+  for (const environmentString of environmentStrings) {
+    const environmentEnumValue: DeploymentEnvironment | undefined =
+      DeploymentEnvironment[
+        environmentString.toLowerCase() as keyof typeof DeploymentEnvironment
+      ]
+
+    if (environmentEnumValue !== undefined) {
+      environmentArray.push(environmentEnumValue)
```

Comment:
We might want to make sure this is unique to stop having 2 environments pointing to the same?